.github/workflows/osx.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: macOS10.14 build and test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  OSX:
+    runs-on: macOS-10.14
```

Comment:
We should parameterize this and test with at least `macos-latest` as well.

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategy

Reply:
added